how-to/how-to-use-sysctls-with-kata.md

```diff
@@ -0,0 +1,128 @@
+# Setting Sysctls with Kata
+
+## Sysctls
+In Linux, the sysctl interface allows an administrator to modify kernel 
+parameters at runtime. Parameters are available via the /proc/sys/ virtual 
+process file system. 
+
+The parameters include the following subsystems among others:
+- kernel (kernel.)
+- networking (net.)
+- virtual memory (vm.)
+- file systems (fs.)
+
+To get a complete list of kernel parameters, you can run:
+```
+sudo sysctl -a
+```
+
+Both Docker and Kubernetes provide mechanism for setting namespaced sysctls. 
+Namespaced sysctls can be set per pod in case of Kubernetes or per container
+in case of Docker.
+The following sysctls are known to be namespaced and can be set with 
+Docker and Kubernetes:
+
+- kernel.shm*,
+- kernel.msg*,
+- kernel.sem,
+- fs.mqueue.*,
+- net.*.
+
+### Namespaced Sysctls:
+
+Kata Containers supports setting namespaced sysctls with Docker and Kubernetes.
+All namespaced sysctls can be set in the same way as regular Linux based
```

Comment:
s/Linux based/namespace based/? or 'software based'.

Reply:
This has to be namespaced sysctls @grahamwhaley. I am referring to sysctls that have a different value per namespace.